lib/gollum/app.rb

```diff
@@ -22,6 +22,13 @@
 # Fix to_url
 class String
   alias :upstream_to_url :to_url
+
+  unless defined?(Grit)
```

Comment:
> that doesn't ensure that gollum-lib will not fall back on grit anyway (when GIT_ADAPTER cannot be required)

Well, it looks like that code will throw a LoadError if `Gollum::GIT_ADAPTER` is defined but requiring the corresponding adapter fails. But you are right that if `GIT_ADAPTER` isn't defined, then grit will be used (for now). But what about 
```ruby
unless ! is_defined?(Gollum::GIT_ADAPTER) || Gollum::GIT_ADAPTER == 'grit'
```